portfolio/src/main/webapp/index.html

```diff
@@ -54,27 +54,27 @@ <h3>My Interests</h3>
           <div id="moving-dot" class="main-dot"><p>Present</p></div>
       
           <div class="timeline-branch" id="astronomy">
-            <img src="/images/jupiter.png" alt="Astronomy" style="left: 1%; top: 5%; border-radius: 50%;" onclick="displayMoreInfo(event, 'astronomyinfo')">
+            <img src="/images/jupiter.png" alt="Astronomy" style="left: 1%; top: 3%; border-radius: 50%; width: 20%" onclick="displayMoreInfo(event, 'astronomyinfo')">
```

Comment:
I haven't seen the displayed page, but have you considered having the same style for all these timeline images.

Reply:
The images are supposed to appear at different locations, so I specified their positional style elements inline with their HTML. Is it better to move this to style.css, even if they all have different styles?